.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
[ OPTIONAL ] You have done an excellent job with your project but I would suggest also ignoring the `node_modules` folder for better practices because sometimes by mistake it could be uploaded and it is not necessary.